### PR TITLE
Benj9250/rpk reload

### PR DESCRIPTION
**Pull request by bneukom**

### CODE REVIEW

File: VitruvioHost/Plugins/Vitruvio/Source/Vitruvio/Private/VitruvioModule.cpp [hunk 1]
```diff
@@ -99,8 +102,9 @@ class FLoadResolveMapTask
 			const std::wstring AbsoluteRpkPath(TCHAR_TO_WCHAR(*FPaths::ConvertRelativePathToFull(RpkPath)));
 			const std::wstring AbsoluteRpkFolder(TCHAR_TO_WCHAR(*FPaths::Combine(FPaths::GetPath(FPaths::ConvertRelativePathToFull(RpkPath)),
 																				 FPaths::GetBaseFilename(UriPath, true) + TEXT("_Unpacked"))));
-			const std::wstring RpkFileUri = prtu::toFileURI(AbsoluteRpkPath);
 
+			const std::wstring RpkFileUri = prtu::toFileURI(AbsoluteRpkPath);
+			IFileManager::Get().DeleteDirectory(AbsoluteRpkFolder.c_str(), false, true);
```
Comment (bstegerESRI) [inline VitruvioHost/Plugins/Vitruvio/Source/Vitruvio/Private/VitruvioModule.cpp:107]: Would it be possible to delete the RPK file after it has been unzipped?

**Review by bstegerESRI — APPROVED** [review]
Very nice!

File: VitruvioHost/Plugins/Vitruvio/Source/Vitruvio/Private/VitruvioModule.cpp (hunk 1 above)
Comment (bneukom) [reply VitruvioHost/Plugins/Vitruvio/Source/Vitruvio/Private/VitruvioModule.cpp:108]: Good point! Added with c9febb8

**Comment by bneukom** [comment]
Thanks for review. Also used full RPK asset path in unpacked folder, otherwise two RPKs with the same name in different folders in  might override each other during unpacking in the temporary folder on the filesystem with commit 
b6b4c60. I'll merge.

**Pull request merged**